tools/sof-kernel-log-check.sh

```diff
@@ -35,9 +40,9 @@ ignore_str="$ignore_str"'|error: debugfs write failed to idle -16'
 #echo "run $0 with parameter '$*' for check kernel message error"
 
 if [ "$ignore_str" ]; then
-    err=$(eval $cmd|grep 'Call Trace' -A5 -B3)$(eval $cmd | grep $project_key | grep -E "$err_str"|grep -vE "$ignore_str")
+    err=$(eval $cmd|grep 'Call Trace' -A5 -B3)$(eval $cmd | grep -E "$err_str"|grep -vE "$ignore_str")
 else
-    err=$(eval $cmd|grep 'Call Trace' -A5 -B3)$(eval $cmd | grep $project_key | grep -E "$err_str")
+    err=$(eval $cmd|grep 'Call Trace' -A5 -B3)$(eval $cmd | grep -E "$err_str")
```

Comment:
The `eval` seems unnecessary, can you try to remove it?

Reply:
Without it, I'm seeing errors.  How about handling this when we switch to journalctl
```
sed: -e expression #1, char 1: unknown command: `''
sed: -e expression #1, char 1: unknown command: `''

```

Reply:
OK forget it for now; one change at a time.